CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 [comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"
 
+## 2024-02-08
+| What's new | |
+| :--------- | :-: |
+| For Studio Enterprise teams, [Schema Change Proposals](https://apollographql.com/blog/graphos-schema-proposals-a-principled-development-workflow-for-graphql-api-platforms) are now available to help teams collaborate on schema changes before laying down code! | ![image](https://github.com/apollographql/apollo-studio-community/assets/743976/ce00e365-316a-49f1-aec7-eef3fa95b8e1)
```

Comment:
I worry we're setting a precedent that every changelog entry should include a `!`. 

```suggestion
| For Studio Enterprise teams, [Schema Change Proposals](https://apollographql.com/blog/graphos-schema-proposals-a-principled-development-workflow-for-graphql-api-platforms) are now generally available to help teams collaborate on schema changes before laying down code. | ![image](https://github.com/apollographql/apollo-studio-community/assets/743976/ce00e365-316a-49f1-aec7-eef3fa95b8e1)
```

Reply:
good point, i vaguely recall someone's style guide pointing out something like this